Neos.Flow/Classes/Command/RoutingCommandController.php

```diff
@@ -37,7 +37,7 @@ class RoutingCommandController extends CommandController
 
     /**
      * @Flow\Inject
-     * @var Router
+     * @var RouterInterface
```

Comment:
The RoutingCommandController invokes methods on the router that are not part of the interface, namely `getRoutes()`